lib/index.js

```diff
@@ -179,4 +193,21 @@ module.exports = class EyeglassCompiler extends BroccoliSassCompiler {
 
     return key + "+" + dependencies;
   }
+
+  cacheAssetImports(collection) {
+    let self = this;
+    let realAsAssetImport = collection.asAssetImport;
+    collection.asAssetImport = function(name) {
```

Comment:
In general, monkey patching something from another library isn't a great idea. Are their any reasonable alternatives?

Reply:
Added an optional function in https://github.com/sass-eyeglass/eyeglass/pull/180 to provide a cleaner alternative for this